log/README.md

```diff
@@ -1,54 +1,45 @@
+
+
 # package log
 
-`package log` provides a minimal interface for structured logging in services.
-It may be wrapped to encode conventions, enforce type-safety, etc.
-It can be used for both typical application log events, and log-structured data streams.
+    import "/Users/harlow/code/go/src/github.com/go-kit/kit/log"
 
 ## Rationale
 
-TODO
+Package log provides basic interfaces for structured logging.
+
+The fundamental interface is Logger. Loggers create log events from
+key/value data.
+
+
 
 ## Usage
 
-Typical application logging.
```

Comment:
Is there a way to distinguish different doc comment "areas" so we can have both usage and rationale sections? If not, let's just combine into one, and put it directly under the main "package log" header, no sub-headers.